pep-0623.rst

```diff
@@ -12,13 +12,13 @@ Abstract
 ========
 
 PEP 393 deprecated some unicode APIs, and introduced ``wchar_t *wstr``,
-and ``Py_ssize_t wstr_length`` in the Unicode strucutre to support
+and ``Py_ssize_t wstr_length`` in the Unicode structure to support
 these deprecated APIs. [1]_
 
 This PEP is planning removal of ``wstr``, and ``wstr_length`` with
 deprecated APIs using these members by Python 3.12.
 
-Deprecated APIs doesn't use the members are out of scope because
+Deprecated APIs which doesn't use the members are out of scope because
```

Comment:
```diff
-Deprecated APIs which doesn't use the members are out of scope because
+Deprecated APIs which don't use the members are out of scope because
```